package.json

```diff
@@ -85,7 +85,7 @@
 		"husky": "^3.0.0",
 		"minimist": "^1.2.2",
 		"mocha": "^9.2.0",
-		"nodemon": "^2.0.15",
+		"nodemon": "^3.1.7",
```

Comment:
[Changelog](https://github.com/remy/nodemon/releases)

Reply:
Breaking changes don't look like an issue, support for Node 8 is dropped: 
https://github.com/remy/nodemon/releases/tag/v3.0.0